src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Link, RouteComponentProps} from 'react-router';
+
+import ResultGrid from 'app/components/resultGrid';
+import {t} from 'app/locale';
+
+type Props = RouteComponentProps<{}, {}>;
+
+const getRow = (row: any) => [
+  <td key={row.id}>
+    <strong>
+      <Link to={`/${row.slug}/`}>{row.name}</Link>
+    </strong>
+    <br />
+    <small>{row.slug}</small>
+  </td>,
+];
+
+const AdminOrganizations = (props: Props) => (
+  <div>
+    <h3>{t('Organizations')}</h3>
+    <ResultGrid
+      path="/manage/organizations/"
+      endpoint="/organizations/?show=all"
+      method="GET"
+      columns={[<th key="column-org">Organization</th>]}
```

Comment:
```suggestion
      columns={[<th key="column-org">{t('Organization')}</th>]}
```

Reply:
@priscilawebdev We don't really translate admin views.